easylinks/src/main/java/com/google/step/servlets/ManageServlet.java

```diff
@@ -1,6 +1,8 @@
 package com.google.step.servlets;
 
 import com.google.step.servlets.ServletHelper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
```

Comment:
Consider removing unnecessary libraries

Reply:
Oh I added that when trying to throw EntityNotFoundException and forgot to remove. Done.